test_scripts/Polices/External_UCS/019_ATF_P_TC_PTU_without_disallowed_by_external_consent_entities_on_struct.lua

```diff
@@ -0,0 +1,120 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PTU without "disallowed_by_external_consent_entities_on" struct
+--
+-- Description:
+-- In case:
+-- SDL receives PolicyTableUpdate without “disallowed_by_external_consent_entities_on:
+-- [entityType: <Integer>, entityId: <Integer>]” -> of "<functional grouping>"
+-- -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PTU as valid (with the pre-conditions of all other valid PTU content)
+-- b. do not create this "disallowed_by_external_consent_entities_on" field
+-- of the corresponding "<functional grouping>" in the Policies database.
+--
+-- Preconditions:
+-- 1. Start SDL (make sure 'disallowed_by_external_consent_entities_on' section is omitted in PreloadedPT)
```

Comment:
@istoimenova, According to question 'Merging rules for "functional_groupings" section and "disallowed_by_external_consent_entities_on /_off" (data exists)' script has to be updated